packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/package.json

```diff
@@ -11,18 +11,15 @@
     "exports": {
       "./package.json": "./package.json",
       ".": "./src/index.ts",
+      "./models": "./src/models/index.ts",
       "./loadTestAdministration": "./src/loadTestAdministration/index.ts",
       "./loadTestAdministration/api": "./src/loadTestAdministration/api/index.ts",
-      "./loadTestAdministration/models": "./src/loadTestAdministration/models/index.ts",
```

Comment:
is this a bugfix considering we have no subClient/models/index generated?

maybe no priority but feel like it's better to have a cadl-ranch case to cover that otherwise we don't know the models are not exported.

Reply:
It is a bugfix, but we can't leverage cadl ranch because we didn't checkin the package.json in cadl ranch.